lib/animina_web/live/profile_live.ex

```diff
@@ -283,33 +282,35 @@ defmodule AniminaWeb.ProfileLive do
   defp fetch_green_and_red_flags(user, language) do
```

Comment:
I don't like the name of the function compared to the result of the function. Same with variables within the function. As far as I understand the code it is not `green_flags` but `green_flags_ids`.